tests/test_tictactoe.py

```diff
@@ -10,6 +10,23 @@ def tt():
     return tictactoe.TicTacToeEnv()
 
 
+def test_policy_page_lines():
+    """
+    Just a calling a policy to make sure that it replay from a couple of different boards.
+    """
+    game = tictactoe.TicTacToeEnv()
+    while not game.done:
+        game.step(random.choice(game.valid_actions))
+        if not game.done:
+            game.step(tictactoe.policy_page_lines(game._get_obs()))
+
+    game = tictactoe.TicTacToeEnv()
```

Comment:
issue: In general, tests should test 1 thing at a time, as much as possible. In this case, the two different orders are unique tests and should be split up.

Reply:
yep, fair enough!